syntax = "proto2";

package BackupPB;

message MachinePath {
	required string machineID = 1;
	required string path = 2;	
}

